Bottom: eaec9d68b55084443c338fabc0e31e7269c1eaf5
Top:    169d48788fd8587e4cb616027239e8ea997b02f8
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 15:14:09 +0000

Adjust STEP export to 2.3mm board thickness (Hack - need something quick!)


---

diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index c736af7..5dafc05 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -31,7 +31,7 @@
 
 #define CRASH fprintf(stderr, "HID error: pcb called unimplemented STEP function %s.\n", __FUNCTION__); abort()
 
-#define HACK_BOARD_THICKNESS MM_TO_COORD(1.6)
+#define HACK_BOARD_THICKNESS MM_TO_COORD(2.3)
 
 static int step_set_layer (const char *name, int group, int empty);
 static void use_gc (hidGC gc);
